[WIP][Iceberg]Support s3 path as warehouse dir for hive and hadoop catalog (#24221)
prestodb/presto: presto-docs/src/main/sphinx/connector/iceberg.rst

                                                        Otherwise, it will be ignored.
======================================================= ============================================================= ============

> By configuring the `Amazon S3 <https://prestodb.io/docs/current/connector/hive.html##amazon-s3-configuration>`_

Reviewer: ```suggestion
Configure the `Amazon S3 <https://prestodb.io/docs/current/connector/hive.html##amazon-s3-configuration>`_
```
Imperative voice suggestion, to make it clear to the reader what they can do to get the result.

> properties, we can specify a S3 location as the warehouse dir of Hadoop catalog. This way, both metadata and data

Reviewer: ```suggestion
properties to specify a S3 location as the warehouse directory for the Hadoop catalog. This way, both metadata and data
```

> of iceberg tables will be maintained on S3 storage. An example configuration includes:

Reviewer: ```suggestion
of Iceberg tables are stored in S3. An example configuration includes:
```
